50.mender-binary-delta/docs.md

```diff
@@ -7,6 +7,43 @@ shortcode-core:
 github: false
 ---
 
+#### mender-binary-delta (1.4.0)
```

Comment:
Can you also add this above:

```diff
+## mender-binary-delta (1.4.0)
+
+_Released 07.06.2022_
+
+### Changelogs
+
 #### mender-binary-delta (1.4.0)
```

I know, it seems a bit redundant here, but it makes sense when compared to the other changelogs, and it also lines up better with the existing content in this document.

Reply:
@kacf done